<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1096</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>20</y>
      <width>841</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QTableView" name="field">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTableView" name="elem1">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="elem2">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="elem3">
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>160</height>
          </size>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>20</number>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1096</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew_game"/>
    <addaction name="actionSettings"/>
    <addaction name="actionHelp"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_game">
   <property name="text">
    <string>New game</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
